articles/connections/references/apple-native/guides/add-siwa-to-web-app.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Add Sign In with Apple to Your Web App
+description: Learn how to add native login functionality to your web app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Your Web App
+
+To set up and configure Sign In with Apple for your web app, you will do the following:
+
+* [Complete the prerequisites](#prerequisites) 
+* [Register your app in your Apple Developer account](#register-your-app-in-your-apple-developer-account)
+* [Configure the application connection in Auth0](#configure-and-test-the-connection-in-Auth0)
+
+See the [Web Application Example](#web-application-configuration-example) for details on how to configure your web application to work with Sign In with Apple in Auth0. 
+
+## Prerequisites
+
+Before you add support for SIWA to your web app, you'll need:
+
+* An [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/)).
+* A domain (such as <YOUR CUSTOM DOMAIN>.com) to point to and an internet-accessible server where you will run the app that responds on behalf of this domain. You will also need to configure this server with a TLS certificate (Apple won't accept unsecured HTTP connections). To use the Email Relay Service, you will also need to configure your domain with Sender Policy Framework (SPF) DNS TXT records.
+
+::: note
+You can set this up using [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04), [Freenom](https://freenom.com/), or [Let's Encrypt](https://letsencrypt.org/).
+:::
+
+* A [Custom Domain](/custom-domains) set up on your Auth0 tenant for domain verification with Apple. Custom domains are not strictly required to use SIWA authorization. Domain validation is required for sending emails to private Apple addresses in native and web apps. 
+
+## Register your web app in your Apple Developer account
+
+When setting up your application with your Apple Developer account, make sure you save the following IDs and keys for the application connection settings in the Auth0 Dashboard:
+
+* Client ID (the Service ID)
+* App ID
```

Comment:
Don't need an App ID for a web app.  You will need the client secret signing key and (optionally) the key identifier)

Reply:
Well... let me walk that back a bit too.  You don't need *strictly* need an App ID for Auth0 configuration, and the only reason anyone ever has to have one is because Apple asks you to associate a Service Identifier with an App ID.  Still uncertain as to the the necessity of that link on the Apple side.